src/sdk/pynni/nni/msg_dispatcher.py

```diff
@@ -136,7 +136,6 @@ def handle_add_customized_trial(self, data):
         # data: parameters
         id_ = _create_parameter_id()
         _customized_parameter_ids.add(id_)
```

Comment:
better to comment that this member func is not supported

Reply:
The hyphen in its name indicates that this is a private method and is not exposed to end users.

Reply:
This method does work. However the protocol has been changed so that this function should not send `NewTrialJob` command back to NNI manager.

Reply:
I should have documented this change in IPC protocol, but the protocol itself is not well documented I have not fully understood it yet.
I'm planning on clean up the protocol and write doc for it.

Reply:
@liuzhe-lz please add an issue for this